### ISSUE TITLE
State support

### ISSUE THREAD

**Issue by badvision** [libretro/nxengine-libretro#38]
I've played through the game on the Genesis port and have noticed the game is kinda fun when you can rewind.  Makes it more like Braid. ;)  I recognize the difficulty in adding stateful support but it would make some moments of the game a tad less frustrating when trying to explore the whole game and discover all the side quests.

**Comment by Darknior** [comment]
Yes if it is possible to have them, it can be really cool. Some part of game are really hard, and hardest for little children's. Save state can make the game more enjoyable for them.

**Comment by crocket** [comment]
I think nxengine should have been ported to PSP or dreamcast or Wii instead of libretro. Game console emulators support states effortlessly.
